frontend/src/components/dialog/DialogBase.vue

```diff
@@ -61,6 +61,9 @@ export default {
         if (e.response.status === 409 /* Conflict */) {
           this.error = this.$tc('global.serverError.409')
         }
+        if (e.response.status === 422 /* Validation Error */) {
+          this.error = e
```

Comment:
Müsste man hier nicht zumindest den Key oder Value der Übersetzung rausholen? Oder noch besser gleich mit `this.$tc` übersetzen

Reply:
Bin ich unsicher... müssen wir diskutieren.